spec/helper_spec.rb

```diff
@@ -171,6 +172,111 @@ def to_s
           expect(trimmed_value[:events].first[:metaData]).to be_nil
         end
       end
+
+      context "and new trimming priorities are in place" do
```

Comment:
This shouldn't be a new context because no conditions are being changed.

Reply:
It's mostly a readability context for me at the moment before the tests are concreted.